charts/presto/templates/configmap-coordinator.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-coordinator
+  labels:
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app.kubernetes.io/name: {{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Values.image.tag | default .Chart.AppVersion | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+  node.properties: |-
+    node.environment={{ .Values.node.environment }}
+    node.data-dir={{ .Values.node.dataDir }}
+  jvm.config: |-
+    {{- .Values.coordinator.jvm | default .Values.jvm | nindent 4 }}
+  config.properties: |-
+    coordinator=true
+    http-server.http.port={{ .Values.service.port }}
+    discovery.uri=http://localhost:{{ .Values.service.port }}
+    discovery-server.enabled=true
+    node-scheduler.include-coordinator={{ eq .Values.mode "single" }}
```

Comment:
Could you also add the following both for worker and coordinator so that we've got it ready for `Prestissimo` testing:
```yaml
  {{ if .Values.worker.usePrestissimoWorkers }}
    optimizer.optimize-hash-generation=false
    inline-sql-functions=false
    use-alternative-function-signatures=true
    parse-decimal-literals-as-double=true
    regex-library=RE2J
    offset-clause-enabled=true
    deprecated.legacy-date-timestamp-to-varchar-coercion=true
  {{ end }}
```

Reply:
It is possible to set the values through `config` property in `values.yaml` file, so it's ready for `Prestissimo` testing.

Reply:
I will assume that a compromise would be moving this discussion to next PR - after this one is being merged as a starting point :). It will also require adding different `Deployment` template etc.